config.json

```diff
@@ -317,6 +317,17 @@
         "char"
       ]
     },
+    {
+      "uuid": "9de405e1-3a05-43cb-8eb3-00b81a2968e9",
+      "slug": "series",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 4,
```

Comment:
If we use a simpler, function-based interface, there's likely a one-line solution using [`Vec::windows`](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.windows), which makes me question the 4 difficulty here a bit.

If we use the struct-based interface, most of the complexity of the exercise comes from implementing the struct itself, which isn't that interesting.

Reply:
👍

Reply:
I changed difficulty to 1 because there are only the difficulties 1, 4, 7, and 10 in use (according to the last point in https://github.com/exercism/rust/blob/master/README.md#contributing-a-new-exercise).